index.md

```diff
@@ -3,15 +3,80 @@ layout: lesson
 root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
-FIXME: home page introduction
+# About the Lesson
+In this lesson, we will work on to focus on various accelerating modules that are implemented in LAMMPS for achieving a performance gain. First, we will discuss about the various options and then I'll show you an example to demonstrate how to decide on input parametrs to gain a performance boost. In the practical session, you will get a chance to work on another LAMMPS problem to test some of the accelearting tools that are embedded in LAMMPS. 
 
-<!-- this is an html comment -->
+The content of the lesson is as follows:
+
+1. What is benchmark?
+    * general definition
+    * real-world example of benchmarking: assessment of understanding
+    * Ask users to build a mental model to link between steps of benchmarking
```

Comment:
Users won't know what you mean by a "mental model", building that model is the instructors job.